updatecli/updatecli.d/update-k8s-k3s/autodiscovery-k8s-deps.yaml

```diff
@@ -0,0 +1,23 @@
+name: '{{ .k8s_k3s.manifest }}'
+pipelineid: '{{ .k8s_k3s.pipelineid }}'
+
+autodiscovery:
+  scmid: 'default'
+  actionid: 'rancher-pr'
+  groupby: all
+  crawlers:
+    golang:
+      versionfilter:
+        kind: semver
+        pattern: '{{ .k8s.deps }}'
+      only:
+        - modules:
+            '^k8s\.io\/.*': ''
+      ignore:
+        - modules:
+            '^k8s\.io\/client-go': ''
+            '^k8s\.io\/kubernetes': ''
+            '^k8s\.io\/kube-openapi': ''
+            '^k8s\.io\/legacy-cloud-providers': ''
+            '^k8s\.io\/helm': ''
+            '^k8s\.io\/utils': ''
```

Comment:
What is the plan for updating the modules, except for k8s.io/kubernetes?

Reply:
The plan here is to add, on subsequent updatecli improvements, an `autodiscovery` workflow to do only patch level bumps for the Go dependencies. On purpose I excluded those ones from the K8s/K3s patch bump, because they don't follow the same release cadence and versioning of K8s -> v1.x.y -> v0.x.y. So, those will be treated in the same way that we should treat all the other Go dependencies, i.e., bump to the last patch version available.

Reply:
SGTM, Thanks for the explanation.